src/igvfd/schemas/lab.json

```diff
@@ -56,14 +64,22 @@
                 "comment": "See award.json for available identifiers.",
                 "type": "string",
                 "linkTo": "Award"
+            },
```

Comment:
Think this should be switched to array- but also Lab can only be submitted by admin, so perhaps not necessary to include examples?